packages/README.md

```diff
@@ -0,0 +1,11 @@
+# Coding agreements in DevTools
```

Comment:
We should link to Effective Dart and the Flutter style guide here:
e.g. ordering guide described here: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#order-other-class-members-in-a-way-that-makes-sense